examples/stripe_webhook_handler.py

```diff
@@ -27,6 +27,7 @@ def webhook():
             # CHECK: fetch_object is present and callable, returning a strongly-typed object (without casting)
```

Comment:
not part of this PR, but this line probably shouldn't be in examples?

Reply:
correct, will fix.  thanks!